package.json

```diff
@@ -44,7 +44,10 @@
     "@types/mocha": "5.2.6",
     "@types/nock": "9.3.1",
     "@types/node": "11.11.4",
-    "algosdk": "1.2.0",
+    "@babel/cli": "^7.4.4",
```

Comment:
Why are we bringing in babel as a dependency?

Reply:
This dependency will likely be removed. Currently it's used by our `algosdk-bitgo` library. We transpile the official `algosdk` so it can be used by the platform.

We'll figure this out before this PR is ready to be merged. I added the DO NOT MERGE label for now.

Reply:
Ok, it's gone now